config/src/networks.rs

```diff
@@ -14,13 +14,14 @@ use crate::base::BaseConfig;
 use crate::types::{ChainConfig, Fork, Forks};
 
 #[derive(
-    Debug, Clone, Copy, Serialize, Deserialize, EnumIter, Hash, Eq, PartialEq, PartialOrd, Ord,
+    Debug, Clone, Serialize, Deserialize, EnumIter, Hash, Eq, PartialEq, PartialOrd, Ord,
 )]
 pub enum Network {
     MAINNET,
     GOERLI,
     SEPOLIA,
     HOLESKY,
+    DEVNET(String),
```

Comment:
What is the string meaning? document

Reply:
if this is the path to the config file, I think that's a very weird place to put it.